libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -264,6 +272,7 @@ static MQTTPublishInfo_t incomingInfo;
  * Session present response is obtained from the CONNACK from broker.
  */
 static void establishMqttSession( MQTTContext_t * pContext,
+                                  bool initContext,
```

Comment:
1. This parameter name doesn't match the comment or the one in the function definition, `initializeContext`.
2. Is this even necessary? Whenever `establishMqttSession` is called, this flag seems to have the same value as `createCleanSession`

Reply:
Good point, can use the `createCleanSession` flag instead